api4/group.go

```diff
@@ -575,28 +575,27 @@ func getGroups(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.Err = model.NewAppError("Api4.getGroups", "api.ldap_groups.license_error", nil, "", http.StatusNotImplemented)
 		return
 	}
+	teamID := c.Params.NotAssociatedToTeam
+	channelID := c.Params.NotAssociatedToChannel
```

Comment:
This method requires either `teamID` or `channelID` right? or both? could just do something like if either is required
```
	if len(teamID) != 26 && len(channelID) != 26 {
		c.Err = model.NewAppError("Api4.getGroups", "api.ldap_groups. invalid_or_missing_channel_or_team_id", nil, "", http.StatusBadStatus)
		return
	}
```

Reply:
Thanks! Done.